src/components/typography/Title.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import typography from "@/themes/typography";
+import styled from "styled-components";
+
+const { title } = typography;
+
+const Title = styled.h2(({ className, subtitle }) => [
+  title.default.base,
+  title.default.color,
+  title.default.size,
+  subtitle && title.subtitle.base, // CAN YOU PLEASE MAKE
+  subtitle && title.subtitle.color, // THESE THREE LINES
+  subtitle && title.subtitle.size, // DRYer?  THANKS.
```

Comment:
Please note the comment here about making these DRYer (if possible).

Reply:
It seems impossible to me

Reply:
@MaksymTaratuta55 

Can you please remove these comments also?  Thanks.